docs/reference/sensors/hwpc-sensor.md

```diff
@@ -149,8 +188,6 @@ In order to run the Sensor without a configuration file, run HWPC Sensor using o
     -c "core" -e "CPU_CLK_THREAD_UNHALTED:REF_P" -e "CPU_CLK_THREAD_UNHALTED:THREAD_P" -e "LLC_MISSES" -e "INSTRUCTIONS_RETIRED"
     ```
 
-
```

Comment:
Suggestions: 
- Provide the same code but using `csv` as ouput
- Provide a sh script for each processor family including launching mongodb/csv and the sensor wih correct events names